dallinger/recruiters.py

```diff
@@ -727,6 +736,60 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, participant: Participant):
+        """Screen-out a participant.
+
+        Parameters:
+            participant: Participant
+                The participant to screen-out.
+        """
+        if self.screen_out_allowed(participant):
+            try:
+                return self.prolificservice.screen_out(
+                    study_id=self.current_study_id,
+                    submission_id=participant.assignment_id,
+                    bonus_per_submission=participant.base_pay + participant.bonus,
+                    increase_places=self.config.get("auto_recruit"),
+                )
+            except ProlificServiceException as ex:
+                logger.exception(str(ex))
+
+    def screen_out_allowed(self, participant):
+        """Check if a participant can be screened out.
+
+        Returns False if:
+        - participant is not in 'working' status
+        - custom screening is enabled
+        - reward is below minimum required
+        """
+        if participant.status != "working":
+            return False
+
+        study = self.prolificservice.get_study(self.current_study_id)
+
+        if study.get("is_custom_screening", False):
```

Comment:
Shouldn't this be the opposite? According to the Prolific documentation, this only works if `is_custom_screening=True`.

Reply:
Yes, found out about that, too. It is fixed.